pyscitt/pyscitt/cli/retrieve_signed_claims.py

```diff
@@ -52,11 +53,22 @@ def cli(fn):
         type=Path,
         help="Folder containing JSON parameter files of SCITT services to trust",
     )
+    parser.add_argument(
+        "--dont-embed-receipt",
+        action="store_true",
+        default=False,
+        help="Whether to not return the signed claimsets with the receipt embedded. Default is False.",
```

Comment:
```suggestion
        help="Whether to include the receipt in the signed claimset under the unprotected header. Default is True.",
```
Avoiding double negative as the text otherwise is quite confusing to the people new to this